### ISSUE TITLE
Add to Dash's User Contributed repo

### ISSUE THREAD

**Issue by khit** [citruspi/PowerShell.docset#1]
This is a GREAT docset to have! Just found it yesterday and have already used it 10+ times! Thanks! 

Have you considered submitting the docset to [Dash's User Contributed repo](https://github.com/Kapeli/Dash-User-Contributions) so that they show in the Downloads section of Dash Preferences? 
![2015-08-20 at 3 35 pm](https://cloud.githubusercontent.com/assets/114688/9395007/48deb904-4751-11e5-880e-fe0344f3e75d.png)

As far as I can tell, the docset already meets all the [requirements](https://github.com/Kapeli/Dash-User-Contributions#contribute-a-new-docset) (and many of the optional ones). Might be a good way to expose more people to your fantastic work here!

Thanks again for creating this! - Kye

**Comment by kamakazikamikaze** [comment]
I agree with @khit , you should consider submitting this so it's possible to download from the App itself. There are many others who probably haven't been able to see this via Google but would happily download if seen in the list of Docsets
